bundles/org.openhab.binding.systeminfo/pom.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>2.5.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>org.openhab.binding.systeminfo</artifactId>
+
+  <name>openHAB Add-ons :: Bundles :: Systeminfo Binding</name>
+
+<!--
+  Please, remove properties section below as far
+  https://github.com/java-native-access/jna/issues/1054
+  is released and oshi release was updated
+-->
+  <properties>
+    <bnd.exportpackage>com.sun.jna.platform.*;version="5.2.0"</bnd.exportpackage>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.github.oshi</groupId>
+      <artifactId>oshi-core</artifactId>
+      <version>3.13.0</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
```

Comment:
remove this dependency (always provided if embedded)

Reply:
Done